augur/metrics/pull_request/pull_request.py

```diff
@@ -372,7 +372,7 @@ def pull_request_acceptance_rate(self, repo_group_id, repo_id=None, begin_date=N
                     WHERE action = 'merged'
                     AND issues.pull_request IS NOT NULL 
                     AND repo_group_id = :repo_group_id
-                    AND issue_events.created_at BETWEEN :begin_date AND :end_date
```

Comment:
have we tested that the pull request model still works with this change?

Reply:
Same situation here, the model will still work, but I do not know why this change was made for multiple metrics @ccarterlandis

Reply:
See my comment above